apps/nextra/pages/en/build/indexer/common-queries/account-transactions-gt.mdx

```diff
@@ -0,0 +1,35 @@
+import { GraphQLEditor } from 'components/index'
+import { Callout } from 'nextra/components'
+
+# Get Recent Account Transactions
+
+`GetAccountTransactionsDataGt` gets transactions impacting a specific account that are at least as recent as a given transaction version. This can be useful for only checking the most recent transactions. 
+
+<Callout type="info">
+Try it yourself! You can customize the variables at the bottom of the editor.
+</Callout>
+
+<GraphQLEditor
+  query={`query GetAccountTransactionsDataGt($address: String, $limit: Int, $gt: bigint) {
+  address_version_from_move_resources(
+    where: {address: {_eq: $address}, transaction_version: {_gt: $gt}},
```

Comment:
Change this query to be 

```graphql
query GetAccountTransactionsData($address: String, $limit: Int) {
  account_transactions(
    where: { account_address: { _eq: $address } }
    order_by: {transaction_version: desc}
    limit: $limit
  ) {
    transaction_version
    __typename
  }
}
```

Reply:
Updated both queries which were similar to use `account_transactions`.